core/src/main/scala/codecs/package.scala

```diff
@@ -18,31 +18,34 @@ package freestyle.cassandra
 
 import java.nio.ByteBuffer
 
+import cats.MonadError
 import com.datastax.driver.core.{DataType, ProtocolVersion, TypeCodec}
 import com.datastax.driver.core.exceptions.InvalidTypeException
 
 package object codecs {
 
   trait ByteBufferCodec[T] {
```

Comment:
Should we unify the naming of these methods? encode/serialize, decode/deserialize. makes sense?

Reply:
Nice catch, makes sense to me